[2022-07-29 10:54:11,036] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Second_one.first_task scheduled__2021-11-07T00:00:00+00:00 [queued]>
[2022-07-29 10:54:11,050] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Second_one.first_task scheduled__2021-11-07T00:00:00+00:00 [queued]>
[2022-07-29 10:54:11,050] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 10:54:11,050] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-07-29 10:54:11,050] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-29 10:54:11,073] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): first_task> on 2021-11-07 00:00:00+00:00
[2022-07-29 10:54:11,077] {standard_task_runner.py:52} INFO - Started process 2643 to run task
[2022-07-29 10:54:11,080] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Second_one', 'first_task', 'scheduled__2021-11-07T00:00:00+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/Second_One.py', '--cfg-path', '/tmp/tmp2qnom6xv', '--error-file', '/tmp/tmpn149mxv3']
[2022-07-29 10:54:11,081] {standard_task_runner.py:80} INFO - Job 104: Subtask first_task
[2022-07-29 10:54:11,161] {task_command.py:369} INFO - Running <TaskInstance: Second_one.first_task scheduled__2021-11-07T00:00:00+00:00 [running]> on host 9f0f006fc89d
[2022-07-29 10:54:11,276] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yosr
AIRFLOW_CTX_DAG_ID=Second_one
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-07T00:00:00+00:00
[2022-07-29 10:54:11,277] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-29 10:54:11,278] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hello world']
[2022-07-29 10:54:11,293] {subprocess.py:85} INFO - Output:
[2022-07-29 10:54:11,296] {subprocess.py:92} INFO - hello world
[2022-07-29 10:54:11,296] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-29 10:54:11,339] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Second_one, task_id=first_task, execution_date=20211107T000000, start_date=20220729T105411, end_date=20220729T105411
[2022-07-29 10:54:11,379] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-29 10:54:11,425] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
